File: app/src/main/java/org/michaelbel/moviemade/rest/api/service/NETWORKS.java
package org.michaelbel.moviemade.rest.api.service;

import org.michaelbel.moviemade.rest.model.v3.Network;

import io.reactivex.Observable;
import retrofit2.http.GET;
import retrofit2.http.Path;
import retrofit2.http.Query;

public interface NETWORKS {

    @GET("network/{network_id}?")
    Observable<Network> getDetails(
        @Path("network_id") int id,
        @Query("api_key") String apiKey
    );
}

File: app/src/main/java/org/michaelbel/moviemade/data/service/NETWORKS.kt
package org.michaelbel.moviemade.data.service

import io.reactivex.Observable
import org.michaelbel.moviemade.data.entity.Network
import retrofit2.http.GET
import retrofit2.http.Path
import retrofit2.http.Query

interface NETWORKS {

    @GET("network/{network_id}?")
    fun getDetails(
        @Path("network_id") id: Int,
        @Query("api_key") apiKey: String
    ): Observable<Network>

    // getAlternativeNames

    // getImages
}